### PR TITLE
statistical query group and sort sample

### PR DESCRIPTION
**Pull request by tschie**
Shows how to query statistics from a feature table

### CODE REVIEW

**Comment by tschie** [comment]
@john0005 This is ready for review. It's got quite a few edge cases (I think I've covered them) so it may take a while to review.

File: src/main/java/com/esri/samples/featurelayers/statistical_query_group_and_sort/StatisticalQueryGroupAndSortController.java [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+            new StatisticDefinition("POP2007", StatisticType.SUM),
+            new StatisticDefinition("POP2007", StatisticType.AVERAGE),
+            new StatisticDefinition("AGE_5_17", StatisticType.MINIMUM)
+        );
+        groupFieldsListView.getItems().stream().filter(f -> f.getFieldName().equals("SUB_REGION")).collect(Collectors
+            .toList()).get(0).setGrouping(true);
+        orderByTableView.getItems().add(new OrderByField(new QueryParameters.OrderBy("SUB_REGION", QueryParameters.SortOrder
+            .ASCENDING)));
+      }
+    });
```
Comment (john0005) [inline src/main/java/com/esri/samples/featurelayers/statistical_query_group_and_sort/StatisticalQueryGroupAndSortController.java]: Would place an alert here if the feature table doesn't load.
new Alert(Alert.AlertType.ERROR, "").show();

File: src/main/java/com/esri/samples/featurelayers/statistical_query_group_and_sort/StatisticalQueryGroupAndSortController.java [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+import com.esri.arcgisruntime.data.StatisticDefinition;
+import com.esri.arcgisruntime.data.StatisticRecord;
+import com.esri.arcgisruntime.data.StatisticType;
+import com.esri.arcgisruntime.data.StatisticsQueryParameters;
+import com.esri.arcgisruntime.data.StatisticsQueryResult;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+
+public class StatisticalQueryGroupAndSortController {
+
+  public ComboBox<String> fieldNameComboBox;
```
Comment (john0005) [inline src/main/java/com/esri/samples/featurelayers/statistical_query_group_and_sort/StatisticalQueryGroupAndSortController.java]: Would make these @FXML private.

**Review by john0005 — APPROVED** [review]
Looks good. Tried some different things and didn't get it to break. Looks cool!

**Pull request merged**